packages/engine/src/scene/components/MediaComponent.ts

```diff
@@ -187,10 +187,12 @@ export function MediaReactor() {
   const mediaElement = useOptionalComponent(entity, MediaElementComponent)
   const audioContext = getState(AudioState).audioContext
   const gainNodeMixBuses = getState(AudioState).gainNodeMixBuses
+  const rendererAvailable = RendererComponent.useRendererAvailable(Engine.instance.viewerEntity)
```

Comment:
this is unnecessary, in fact, the assumption in the useEffect that we are targeting `Engine.instance.viewerEntity` is part of the problem. we need to use `useRendererEntity` here instead